test/scenarios/el94-src@rpm-install.sh

```diff
@@ -36,37 +36,35 @@ scenario_remove_vms() {
 scenario_run_tests() {
     local -r source_reponame=$(basename "${LOCAL_REPO}")
     local -r source_repo_url="${WEB_SERVER_URL}/rpm-repos/${source_reponame}"
-    local -r target_version=$(current_version)
+    local -r target_version=$(local_rpm_version)
 
     # Enable the repositories with the dependencies.
 
     # RHOCP for current version might not exist yet. Then, just use whatever we can get for previous minor.
-    local -r rhocp_current=$("${ROOTDIR}/scripts/get-latest-rhocp-repo.sh" "${MINOR_VERSION}" || true)
-    local -r rhocp_previous=$("${ROOTDIR}/scripts/get-latest-rhocp-repo.sh" "${PREVIOUS_MINOR_VERSION}")
-    for rhocp in "${rhocp_current}" "${rhocp_previous}"; do
-        if [[ ! "${rhocp}" ]]; then
-            # rhocp_current might be empty if the beta mirror is not up yet
+    for rhocp in "${CURRENT_RELEASE_REPO}" "${PREVIOUS_RELEASE_REPO}" ; do
+        if [[ -z "${rhocp}" ]]; then
             continue
         fi
 
-        if [[ "${rhocp}" =~ ^[0-9]{2} ]]; then
-            run_command_on_vm host1 "sudo subscription-manager repos --enable rhocp-4.${rhocp}-for-rhel-9-\$(uname -m)-rpms"
+        if [[ "${rhocp}" =~ ^rhocp ]]; then
+            run_command_on_vm host1 "sudo subscription-manager repos --enable ${rhocp}"
         elif [[ "${rhocp}" =~ ^http ]]; then
-            url=$(echo "${rhocp}" | cut -d, -f1)
-            ver=$(echo "${rhocp}" | cut -d, -f2)
-            OCP_REPO_NAME="rhocp-4.${ver}-for-rhel-9-mirrorbeta-$(uname -i)-rpms"
+            local ocp_repo_name
+            ocp_repo_name="rhocp-for-rhel-9-mirrorbeta-rpms"
+            local tmp_file
             tmp_file=$(mktemp)
+
             tee "${tmp_file}" >/dev/null <<EOF
-[${OCP_REPO_NAME}]
-name=Beta rhocp-4.${ver} RPMs for RHEL 9
-baseurl=${url}
+[${ocp_repo_name}]
+name=Beta rhocp RPMs for RHEL 9
+baseurl=${rhocp}
 enabled=1
 gpgcheck=0
 skip_if_unavailable=0
 EOF
             copy_file_to_vm host1 "${tmp_file}" "${tmp_file}"
-            run_command_on_vm host1 "sudo cp ${tmp_file} /etc/yum.repos.d/${OCP_REPO_NAME}.repo"
-            rm "${tmp_file}"
+            run_command_on_vm host1 "sudo cp ${tmp_file} /etc/yum.repos.d/${ocp_repo_name}.repo"
```

Comment:
Oh, this is not right.
We want to enable both repositories, but using the same filename we overwrite one with the other, rather than having both.